src/dropdown/dropdown.stories.ts

```diff
@@ -17,6 +17,7 @@ storiesOf("Dropdown", module)
 	.add("Basic", () => ({
 		template: `
 		<div style="width: 300px">
+			<input type="text"/>
```

Comment:
Looks like something used for testing, do we want to keep it here?

Reply:
Oops, yep that definitely shouldn't be there